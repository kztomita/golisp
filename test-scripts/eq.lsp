(print (eq 3 3))
(print (eq 3 (+ 1 2)))
(print (eq 3 2))
(print (eq 'a 'a))
(print (eq 'a 'b))
(print "====")	; list
(print (eq '(a b) '(a b)))
(setq a '(a b))
(print (eq a a))
(print "====")	; sequence
(print (eq "aaa" "aaa"))
(setq a "aaa")
(print (eq a a))
