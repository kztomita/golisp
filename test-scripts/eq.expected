
t 
t 
nil 
t 
t 
nil 
"====" 
nil 
t 
"====" 
nil 
t 
